sources/packages/web/src/views/student/AppStudent.vue

```diff
@@ -9,7 +9,7 @@
         class="navigation-btn float-left"
       >
         <v-btn
-          v-if="hasAuthenticatedStudentAccount"
```

Comment:
As discussed, please continue to use `hasAuthenticatedStudentAccount` and remove `showIndividualMenuButtons`.